plugins/inputs/smart/smart.go

```diff
@@ -136,7 +136,7 @@ func (m *Smart) scan() ([]string, error) {
 
 	devices := []string{}
 	for _, line := range strings.Split(string(out), "\n") {
-		dev := strings.Split(line, " ")
```

Comment:
Although it is more overhead, this should be:
```go
		dev := []string{}
		if strings.Contains(line, "#") {
			dev = strings.Split(line, "#")
		} else {
			dev = strings.Split(line, " ")
		}
```
To handle any case where `smartctl --scan` returns something like:
```
/dev/sda -d scsi
/dev/sdb -d scsi
```